### PR TITLE
feat: add import config option and tests

### LINKED ISSUE #3: Update import to not show warning with Lit 2
Currently there is a warning when using this loader with Lit 2:

```
The main 'lit-element' module entrypoint is deprecated. Please update your imports to use the 'lit' package: 'lit' and 'lit/decorators.ts' or import from 'lit-element/lit-element.ts'."
```

We should change the loader to use `'lit-element/lit-element.js'` which does not trigger the warning for Lit 2.
- https://github.com/lit/lit/blob/main/packages/lit-element/src/lit-element.ts
- https://github.com/lit/lit-element/blob/master/src/lit-element.ts

### LINKED ISSUE #5: Tests
Need 'em

### PR DESCRIPTION
**Pull request by web-padawan**
Fixes #3 
Fixes #5

Based on the suggestion from https://github.com/bennypowers/lit-css-loader/issues/3#issuecomment-835792467

### CODE REVIEW

**Comment by web-padawan** [comment]
@bennypowers Added tests, please take a look when you have time.

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  api.assertVersion(MIN_BABEL_VERSION);
+  api.cache(true);
+
+  return {
+    presets: [
+      [
+        "@babel/preset-env",
+        {
+          targets: {
+            node: "12.22.0",
```
Comment (bennypowers) [inline babel.config.js:13]: why?

**Review by bennypowers — APPROVED** [review]

**Comment by bennypowers** [comment]
This is published under [v0.1.0](https://www.npmjs.com/package/lit-css-loader/v/0.1.0)

Thank you for contributing 🏆

File: babel.config.js (hunk 1 above)
Comment (web-padawan) [reply babel.config.js:13]: I copied the config from css-loader, some parts are probably not needed.

**Pull request merged**